app/components/inputs/SettingsInput.js

```diff
@@ -1,6 +1,5 @@
 import Select from "react-select";
 import { injectIntl } from "react-intl";
-import "style/Input.less";
 
 @autobind
```

Comment:
Do you think we could refactor this to be a functional component as part of this effort ?